admin_guide/sysctls.adoc

```diff
@@ -0,0 +1,168 @@
+[[admin-guide-sysctls]]
+= Sysctls
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+Certain sysctl settings are exposed via Kubernetes, allowing users to modify
+kernel parameters at runtime for namespaces within a container. Only sysctls
```

Comment:
@sttts How's this intro? I'd rather let the progression of the topic work its way to the safe vs unsafe discussion, but hopefully this doesn't mislead right out the gate as much as it did previously?